package.json

```diff
@@ -28,11 +29,21 @@
     "babel-polyfill": "^6.26.0",
     "babel-preset-es2015": "^6.24.1",
     "babel-preset-stage-0": "^6.24.1",
+    "cross-env": "^7.0.2",
+    "css-loader": "^4.3.0",
+    "html-webpack-plugin": "^4.5.0",
+    "mini-css-extract-plugin": "^0.12.0",
+    "react-refresh": "^0.8.3",
     "source-map-loader": "^1.0.1",
+    "style-loader": "^2.0.0",
     "ts-loader": "^8.0.1",
     "typescript": "^3.9.7",
     "webpack": "^4.43.0",
     "webpack-cli": "^3.2.1",
-    "webpack-dev-server": "^3.1.14"
+    "webpack-dev-server": "^3.11.0"
+  },
+  "scripts": {
+    "start": "webpack-dev-server --hot --mode development",
```

Comment:
Can you add `--host 0.0.0.0` so that it is not listening on localhost by default.
Recall our conversation regarding my dev setup where its running in a VM with a specific IP so that I'm not running against localhost.